flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -493,49 +494,110 @@ class FlxTilemapExt extends FlxTilemap
 	}
 	
 	/**
-	 * Sets the tiles that are treated as "clouds" or blocks that are only solid from the top.
+	 * Sets the slope arrays, which define which tiles are treated as slopes.
 	 * 
-	 * @param 	Clouds	An array containing the numbers of the tiles to be treated as clouds.
+	 * @param 	Northwest 	An array containing the numbers of the tiles facing Northwest to be treated as floor tiles with a slope on the left.
+	 * @param 	Northeast	An array containing the numbers of the tiles facing Northeast to be treated as floor tiles with a slope on the right.
+	 * @param 	Southwest	An array containing the numbers of the tiles facing Southwest to be treated as ceiling tiles with a slope on the left.
+	 * @param 	Southeast	An array containing the numbers of the tiles facing Southeast to be treated as ceiling tiles with a slope on the right.
 	 */
-	public function setClouds(?Clouds:Array<Int>):Void
+	public function setSlopes(?Northwest:Array<Int>, ?Northeast:Array<Int>, ?Southwest:Array<Int>, ?Southeast:Array<Int>):Void
 	{
-		if (Clouds != null)
+		if (Northwest != null)
 		{
-			for (i in 0...(Clouds.length))
-			{
-				setTileProperties(Clouds[i], FlxObject.CEILING);			
-			}
+			_slopeNorthwest = Northwest;
+		}
+		if (Northeast != null)
+		{
+			_slopeNortheast = Northeast;
+		}
+		if (Southwest != null)
+		{
+			_slopeSouthwest = Southwest;
+		}
+		if (Southeast != null)
+		{
+			_slopeSoutheast = Southeast;
 		}
+		
+		setSlopeProperties();
 	}
 	
 	/**
-	 * Sets the slope arrays, which define which tiles are treated as slopes.
+	 * Sets the slopes steepness. To set GENTLE and STEEP slopes ThickTiles and ThinTiles are needed and to set MODERATE slopes just need ThickTiles.
 	 * 
-	 * @param 	LeftFloorSlopes 	An array containing the numbers of the tiles to be treated as floor tiles with a slope on the left.
-	 * @param 	RightFloorSlopes	An array containing the numbers of the tiles to be treated as floor tiles with a slope on the right.
-	 * @param 	LeftCeilSlopes		An array containing the numbers of the tiles to be treated as ceiling tiles with a slope on the left.
-	 * @param 	RightCeilSlopes		An array containing the numbers of the tiles to be treated as ceiling tiles with a slope on the right.
+	 * @param 	ThickTiles 	An array containing the numbers of the tiles to be treated as thick slope.
+	 * @param 	ThinTiles	An array containing the numbers of the tiles to be treated as thin slope.
 	 */
-	public function setSlopes(?LeftFloorSlopes:Array<Int>, ?RightFloorSlopes:Array<Int>, ?LeftCeilSlopes:Array<Int>, ?RightCeilSlopes:Array<Int>):Void
+	public function setSteepness(steepness:FlxSlopeSteepness, ThickTiles:Array<Int>, ?ThinTiles:Array<Int>) 
```

Comment:
The default is `FlxSlopeSteepness.MODERATE` but I know what you mean. The reason was to permit the user set the slope back to moderate, and in this case, he would use just `ThickTiles`.
I think slopes will never change angles, so this really is unnecessary.

So I think we don't need the `FlxSlopeSteepness` Enum, and can back to `setGentle()` and `setSteep()` separeted functions

what you think?
`setGentle(ThickTiles:Array<Int>, ThinTiles:Array<Int>)`
`setSteep(ThickTiles:Array<Int>, ThinTiles:Array<Int>)`